mod-source-record-manager-server/src/main/java/org/folio/services/util/EventHandlingUtil.java

```diff
@@ -44,33 +44,40 @@ private EventHandlingUtil() {
   public static Future<Boolean> sendEventToKafka(String tenantId, String eventPayload, String eventType,
                                                  List<KafkaHeader> kafkaHeaders, KafkaConfig kafkaConfig, String key) {
     Event event = createEvent(eventPayload, eventType, tenantId);
-
     String topicName = createTopicName(eventType, tenantId, kafkaConfig);
-
     KafkaProducerRecord<String, String> record = createProducerRecord(event, key, topicName, kafkaHeaders);
 
     String chunkId = extractHeader(kafkaHeaders, "chunkId");
     String recordId = extractHeader(kafkaHeaders, "recordId");
+    String jobExecutionId = extractHeader(kafkaHeaders, "jobExecutionId");
 
     String producerName = eventType + "_Producer";
-    LOGGER.debug("sendEventToKafka:: Starting to send event to Kafka for eventType: {} and recordId: {} and chunkId: {}", eventType, recordId, chunkId);
+    LOGGER.debug("sendEventToKafka:: Starting to send event to Kafka for eventType: {}, jobExecutionId: {}, recordId: {} and chunkId: {}",
+      eventType, jobExecutionId, recordId, chunkId);
 
     KafkaProducer<String, String> producer = createProducer(eventType, kafkaConfig);
     return producer.send(record)
         .eventually(x -> producer.close())
         .map(true)
-        .onSuccess(x -> logSendingSucceeded(eventType, chunkId, recordId))
+        .onSuccess(x -> logSendingSucceeded(eventType, jobExecutionId, chunkId, recordId))
         .recover(err -> handleKafkaPublishingErrors(eventPayload, producerName, eventType, err));
   }
 
-  private static void logSendingSucceeded(String eventType, String chunkId, String recordId) {
+  private static void logSendingSucceeded(String eventType, String jobExecutionId, String chunkId, String recordId) {
     if (recordId == null) {
-      LOGGER.info("logSendingSucceeded:: Event with type: {} and chunkId: {} was sent to kafka", eventType, chunkId);
+      LOGGER.info("logSendingSucceeded:: Event with type: {} for jobExecutionId: {} and chunkId: {} was sent to kafka",
+        eventType, jobExecutionId, chunkId);
     } else {
-      LOGGER.info("logSendingSucceeded:: Event with type: {} and recordId: {} was sent to kafka", eventType, recordId);
+      LOGGER.info("logSendingSucceeded:: Event with type: {} for jobExecutionId: {} and recordId: {} was sent to kafka",
+        eventType, jobExecutionId, recordId);
     }
   }
 
+  public static String extractJobExecutionId(String eventPayload) {
```

Comment:
@Aliaksandr-Fedasiuk  Maybe it would be better to make it private and non-static?
Or if we need it static, move it to the specific Util class.
What do you think?

Reply:
Thanks, this is the unnecessary method. Removed.